pkg/helpers/os/os_linux.go

```diff
@@ -0,0 +1,29 @@
+package os
+
+import (
+	"fmt"
+	"os"
+	"runtime"
+)
+
+// NewOS returns an OS identifier.
+func NewOS() (*OS, error) {
+	variant := "unknown"
+
+	if _, err := os.Stat("/etc/debian_version"); !os.IsNotExist(err) {
```

Comment:
Use `/etc/os-release`, it’s a modern, universal, parseable file format.

Reply:
Ideally yes ! but I would prefer not opening a file every time a task is evaluation which could be trigger at every shell hook evaluation.

If we ever support more than debian that would make sense, for now, we just want to know if apt is supported.

Reply:
This work as is but i'm gonna experiment in a follow up PR with https://github.com/acobaugh/osrelease. stay tuned !